src/mmw/apps/user/views.py

```diff
@@ -47,6 +47,20 @@ def ajax_login(request):
     return Response(data=response_data, status=status)
 
 
+@decorators.api_view(['GET'])
+@decorators.permission_classes((AllowAny, ))
+def ajax_logout(request):
+    response_data = {}
+    status = 200
```

Comment:
I think 200 is the default so, unless you need to send an error code, you should be able to leave this out.